Readthedocs (#117)
bigbio/quantms: docs/dia.rst

The DIA-NN is run with the following parameters than can be changed in the commandline:

- `--missed-cleavages`: Number of missed-cleavages.
- `--min-pep-len & --max-pep-len`: Minimum & Maximum length of the peptides for the search.
- `--min-pr-charge & --max-pr-charge`: Minimum & Maximum charge states.
- `--var-mods`: Maximum number of modifications allows for a peptide.
- `--IL_equivalent`: This parameter is used to handle I/L in the same way or differently. If `--IL_equivalent true` the I/L are considered the same.

**Match between runs (MBR)**: In MBR is allowed, peptides identified by tandem mass spectra in one run are transferred to another by inference based on m/z, charge state, retention time, and ion mobility when applicable. This options is available in the LFQ DDA workflow (:doc:`lfq`). Similarly to the LFQ DDA workflow, the LFQ DIA pipeline allows to perform match between runs by defining `--targeted_only false`.

MSstats
------------

> The output of DIA-NN are exported to Mststas for the downstream analysis, you can read more about MSstats in :doc:`msstats`.

Reviewer: ```suggestion
The output of DIA-NN is exported to MSstats for the downstream analysis, you can read more about MSstats in :doc:`msstats`.
```

Important technical notes
--------------------------

By 2022, the quantms DIA workflow based on DIA-NN has the following drawbacks:

> - **Conda NOT supported**: The quantms pipeline can be only run using **docker**, and **singularity**. The quantms team is working hardly to support also conda, you can follow the `quantms discussions <https://github.com/bigbio/quantms/discussions>`_

Reviewer: ```suggestion
- **Conda NOT supported**: The steps of the DIA branch of the pipeline can only be run using **docker**, and **singularity**. The quantms team is working hard to also support conda. You can follow the `quantms discussions <https://github.com/bigbio/quantms/discussions>`_
```

> - **Parallelization**: The DIA-NN works similar to other tools like MaxQuant, where all the steps of the pipeline are done in one big heavy node. quantms aims is different (see :doc:`lfq`, :doc:`iso`), we aim to parallelize and distribute most of the tasks in the cluster. We are working to develop better ways to parallelize and distribute the tasks in the future.

Reviewer: ```suggestion
- **Parallelization**: The DIA-NN works similar to other tools like MaxQuant, where all the steps of the pipeline are done in one big heavy node. quantms' aims are a bit different (see :doc:`lfq`, :doc:`iso`), we aim to parallelize and distribute most of the tasks in the cluster. We are working to develop better ways to parallelize and distribute the tasks in the future.
```
